Assets/Scripts/SimplePlayerController.cs

```diff
@@ -0,0 +1,32 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SimplePlayerController : MonoBehaviour
+{
+    [SerializeField] private float minForwardSpeed = 3f;
+    [SerializeField] private float maxForwardSpeed = 5f;
+
+    [SerializeField] private float maxSideRotation = 0.02f;
+    [SerializeField] private float maxPitchRotation = 0.01f;
+
+    void Update() => UpdateMovement();
+
+    private void UpdateMovement()
+    {
+        var oldPosition = transform.position;
+
+        var forwardMovement = Input.GetButton("Jump");
+        var mouseY = Input.GetAxis("Mouse Y");
+        var mouseX = Input.GetAxis("Mouse X");
+
+        var newPosition = oldPosition
+            + transform.forward * Time.deltaTime * (forwardMovement ? maxForwardSpeed : minForwardSpeed)
+            + mouseX * transform.right * Time.deltaTime
+            + mouseY * transform.up * Time.deltaTime;
```

Comment:
out of curiosity, would you still get a satisfying result if you only multiplied the final result with Time.deltatime instead of doing it 3 times on the added components?

Reply:
Should be ye! Good point

Reply:
@jmcduff didn't work exactly with multiplying the final result, but here's how it should be done!